controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/screwdriver/ScrewdriverApiTest.java

```diff
@@ -73,7 +73,7 @@ public void testJobStatusReporting() throws Exception {
         notifyCompletion(app.id(), projectId, JobType.systemTest, Optional.empty());
 
         // Notifying about unknown job fails
-        tester.containerTester().assertResponse(new Request("http://localhost:8080/screwdriver/v1/jobreport",
+        tester.containerTester().assertResponse(new Request("http://localhost:8080/application/v4/tenant/tenant1/application/application1/jobreport",
```

Comment:
Should update the other references to `/screwdriver/v1/jobreport` in this test as well so that the new path is tested. When we remove the old path we should move these tests to `ApplicationApiTest`.